package.json

```diff
@@ -99,11 +101,13 @@
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
+    "nebenan-form": "^8.2.1",
     "nebenan-helpers": "^3.6.0 || ^4.1.1",
     "nebenan-react-hocs": "^5.2.2",
     "nebenan-ui-kit": "^3.0.0 || ^4.0.0",
     "node-sass-glob-importer": "^5.3.2",
     "smartcontent": "^3.8.1",
+    "string-validate": "^3.0.3",
```

Comment:
do not forget to remove here as well. Also don't just remove it from package.json. update package-lock.json as well